damus-io/damus
# Translations

## Summary

- Fixed localization issue from AddMuteItemView
- Fix non-breaking spaces in localized strings
- Imports non-English translations from Transifex

## Checklist

- [x] I have added the signoff line to all my commits. See [Signing off your work](../docs/CONTRIBUTING.md#sign-your-work---the-developers-certificate-of-origin)
    - [ ] I do not need to add a changelog entry. Reason: _[Please provide a reason]_
- [x] I have added appropriate `Closes:` or `Fixes:` tags in the commit messages wherever applicable, or made sure those are not needed. See [Submitting patches](https://github.com/damus-io/damus/blob/master/docs/CONTRIBUTING.md#submitting-patches)

## Test report

**Device:** iPhone 16 Pro Simulator

**iOS:** 18.1

**Damus:** a609954ef4801d161938733de2a4c96ee1afeca1

**Setup:** N/A

**Steps:**

`Test Add mute item button`
1. In Xcode, at the top bar underneath the Xcode menu bar, select the target scheme to display the drop down menu.
2. Ensure the `damus` is selected.
3. In the same drop down menu, click `Edit Scheme...`.
4. Click the `Options` tab.
5. Next to `App Language`, select `Accented Pseudolanguage`.
6. Click `Close`.
7. Press Cmd + R to build and run Damus iOS on the simulator.
8. Notice that all the internationalized strings have accented characters.
9. Open the side menu.
10. Tap the `Muted` tab.
11. Tap the `+` button.
12. Observe that the `Add mute item` button has accented characters.

`Test non-breaking spaces fix`
1. In Xcode, press Cmd + R to build and run Damus iOS on the simulator.
2. Go to iOS settings, `Apps` -> `Damus` -> `Language` -> `Spanish (Spain)`
3. Switch back to Damus.
4. Open the side menu.
5. Tap the `Wallet` tab (second tab).
6. Notice that the string that contains the words `Nostr Wallet Connect` is in Spanish.

**Results:**
  - Details: _[Please provide details of the partial pass]_

## Other notes

N/A
